src/components/MainPage/MainPage.js

```diff
@@ -40,6 +40,7 @@ const MainPage = ({
   flowDiagramLastDate,
   flowDiagramPrevDate,
   keyInsightsData,
+  keyInsightsDataJails,
```

Comment:
To make props consistent move `Data` word to the end of the name. `jailsKeyInsightsData` or `keyInsightsJailsData`. I think the better way is to start with DOMAIN (Jails, Corrections) then component `keyInsights`, `flowDiagram`.

Reply:
fixed